test/dom.test.js

```diff
@@ -202,16 +202,34 @@ describe('equalizeNode()', () => {
   ck('collapses spaces inside inline element inside pre',
     '<pre> <span>   \n </span> </pre>',
     '<pre> <span> </span> </pre>');
-  // ck('collapses spaces between inline pre and text at the end of a div',
-  //   '<div>foo</foo><div>   <pre style="display: inline"> hello</pre>  \n \n \t foo </div>',
-  //   '<div>foo</foo><div><pre style="display: inline"> hello</a> foo</div>');
 
-  ck('works with forms',
```

Comment:
@koraa Why did you remove this test ? how do we know if your change fixed the issue I managed to isolate ?